doc/reliability.rst

```diff
@@ -0,0 +1,40 @@
+``mybinder.org`` status and reliability
+=======================================
+
+This page serves as a source of information for general Binder status
+updates, building/launching statistics, and reliability metrics.
+
+Site Reliability Goals
+----------------------
+
+``mybinder.org`` is a free service running on open-source software. It should
```

Comment:
As mybinder.org is a research pilot project, the main goal for the project is to understand usage patterns and workloads for future project evolution. While we strive for site reliability and availability, we want our users to understand the intent of this service is research and we offer no guarantees of its performance in mission critical uses.